frontend/src/theme.ts

```diff
@@ -18,6 +18,13 @@ const theme = createTheme({
     fontFamily: '"Noto Sans", sans-serif',
   },
   components: {
+    MuiDialog:{
+      styleOverrides: {
+        root: {
+          backdropFilter: 'blur(4px)'
+        },
+      },
+    },
```

Comment:
Override Backdrop component instead.

```suggestion
    MuiBackdrop: {
      styleOverrides: {
        root: {
          backdropFilter: 'blur(4px)',
        },
      },
    },
```

Reply:
Updated